atlp-rwanda/e-comm-team-techsmith-fn
# #185475162 Users should be able to order multiple items at once

## What does this PR do?

This PR allows a user to checkout multiple orders at once.

## Description of Task to be completed?

### Why
A user needs to be able to order items in bulk or multiple orders at the same time.

### Acceptance Criteria

User should be able to submit multiple orders at once without using a loop, and the backend should handle them in bulk.

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-multiple-orders-checkout-185475162`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## Any background context you want to provide?

- This PR also improves the Seller 2FA by providing them with short 6-digit codes instead of lengthy tokens for validation.
-  By default, users can order the last three orders that they created.

## What are the relevant pivotal tracker/Trello stories?

- [#185475162](https://www.pivotaltracker.com/story/show/185475162)

## Screenshots (if appropriate)

<img width="1440" alt="Screenshot 2023-06-25 at 02 52 23" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/29fd9fb7-6ec7-4c34-9466-b9d48298f4b3">
<img width="1440" alt="Screenshot 2023-06-25 at 11 39 58" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/c5f4b672-65f5-477d-9021-4617979ee870">
<img width="1440" alt="Screenshot 2023-06-25 at 11 40 09" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/a9413559-0424-4e89-82e0-19065b597414">
<img width="1440" alt="Screenshot 2023-06-25 at 11 41 41" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/aa1834e0-a3f1-4084-bb98-59d9ac700d15">
